server/src/app.js

```diff
@@ -20,12 +21,22 @@ const passport = require('passport')
 const RedisStore = require('connect-redis')(session)
 const redis = require('./redis')
 const elasticsearch = require('./elasticsearch')
+const Raven = require('raven')
 const startSubscriptionServer = require('./graph/subscriptions/SubscriptionServer')
 const PORT = 8181
 
+const GIT_HASH = execSync('git rev-parse HEAD', {
```

Comment:
Agreed, there are a few initialization steps like this that need to be moved to docker image creation, it shouldn't be hard to move over but it hasn't been a priority.